src/containers/VedtakContainer.tsx

```diff
@@ -49,28 +63,38 @@ const VedtakContainer = () => {
 
     return (
         <Side fnr={fnr} tittel={texts.pageTitle} aktivtMenypunkt={VEDTAK}>
-            {vedtak && selectedVedtak &&
-            <>
-                <Row><StyledAlertStripe>{texts.comingSoon}</StyledAlertStripe></Row>
-                <Row>
-                    <Column className="col-xs-5">
-                        {vedtak.data.map((v: VedtakDTO) => (
-                            <StyledPanel>
-                                <Undertittel>
-                                    {restdatoTildato(v.vedtak.fom)} - {restdatoTildato(v.vedtak.tom)}
-                                </Undertittel>
-                                <Undertekst>
-                                    {`Vedtaksdato: ${restdatoTildato(v.opprettet)} · Restdager: ${v.vedtak.gjenståendeSykedager}`}
-                                </Undertekst>
-                            </StyledPanel>
-                        ))}
-                    </Column>
-                    <Column className="col-xs-7">
-                        {vedtak.data.map((vedtak: VedtakDTO) => <VedtakInfopanel selectedVedtak={vedtak}/>)}
-                    </Column>
-                </Row>
-            </>}
-        </Side>);
+            {vedtak && selectedVedtak && (
+                <>
+                    <Row>
+                        <StyledAlertStripe>{texts.comingSoon}</StyledAlertStripe>
+                    </Row>
```

Comment:
Forslag til oppgave når denne er merget: Flytte infoen lenger opp, så man får denne meldingen uansett om det finnes vedtak eller ikke. Jeg tenker det kan være forvirrende for en veileder at siden av og til er helt blank, da ser det ut som at ting ikke lastet ordentlig 🤷‍♂️